cloud_blobstore/__init__.py

```diff
@@ -1,6 +1,64 @@
 import typing
 
 
+class PagedIter(typing.Iterable[str]):
+    """
+    Provide an iterator that will iterate over every object, filtered by prefix and delimiter. Alternately continue
+    iteration with token and key (marker).
+    """
+    def __init__(self):
```

Comment:
don't need this empty constructor.

Reply:
That's fine with me.

Should this empty constructor be discarded as well, or is it there for a reason I'm not seeing?

https://github.com/xbrianh/cloud-blobstore/blob/df6aba3c310a49690691d893d64b6bab5d7aedc4/cloud_blobstore/__init__.py#L64